src/app/features/features.module.ts

```diff
@@ -0,0 +1,10 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { CoursesModule } from './courses/courses.module';
+
+@NgModule({
+  declarations: [],
+  imports: [CommonModule, CoursesModule],
+  exports: [CoursesModule],
+})
+export class FeaturesModule {}
```

Comment:
This module is redundant, remove it and create an index.ts file instead, where:
```
 export * './components/courses-list/courses-list.component.ts';
 export * './components/courses-list/courses-list.module.ts';
```
I also encourage you to create index.ts file in every module or folder, it helps to shorten paths, for example:
from './components/courses-list/courses-list.module.ts' to  './components';